src/model/test/AirTerminalDualDuctVAV_GTest.cpp

```diff
@@ -135,3 +138,22 @@ TEST_F(ModelFixture, AirTerminalDualDuctVAV) {
     }
   }
 }
+
+TEST_F(ModelFixture, AirTerminalDualDuctVAV_MinimumAirFlowTurndownSchedule) {
```

Comment:
Ok, I see 5 model tests. 

* [ ] TODO: I would like to see 5 FT tests too (or modification of existing FT tests) for the new fields.